dev/js/api/api.js

```diff
@@ -1,133 +1,143 @@
-    return STORAGE.getAuthToken({url: url}).then((token) => {
-        return new Promise((resolve, reject) => {
-            let headers = {'X-CLIENT-KEY' : CONFIG.API_CLIENT_KEY,
-                           'X-AUTH-KEY' : CONFIG.API_AUTH_KEY}
-            if (token) headers['X-CID'] = token
-            axiosInstance({
-                method: 'put',
-                url: url,
-                data: data,
-                headers
-            }).then((res) => {
-                resolve(res.data)
-            }, (response) => {
-                let urlInfo = {
-                    url: url,
-                    type: 'API_PUT',
-                    data: data,
-                    token: token
-                }
-                rejectHandler(response, urlInfo, reject)
-            })
-        })
-    })
-
-
-}
+  return STORAGE.getAuthToken({ url: url }).then((token) => {
+    return new Promise((resolve, reject) => {
+      let headers = {
+        'X-CLIENT-KEY': CONFIG.API_CLIENT_KEY,
+        'X-AUTH-KEY': CONFIG.API_AUTH_KEY,
+      };
+      if (token) headers['X-CID'] = token;
+      axiosInstance({
+        method: 'put',
+        url: url,
+        data: data,
+        headers,
+      }).then(
+        (res) => {
+          resolve(res.data);
+        },
+        (response) => {
+          let urlInfo = {
+            url: url,
+            type: 'API_PUT',
+            data: data,
+            token: token,
+          };
+          rejectHandler(response, urlInfo, reject);
+        }
+      );
+    });
+  });
+};
 
 const API_DELETE = (url) => {
-    return STORAGE.getAuthToken({url: url}).then((token) => {
-        return new Promise((resolve, reject) => {
-            let headers = {}
-            if (token) headers['X-CID'] = token
-            axiosInstance({
-                method: 'delete',
-                url: url,
-                headers
-            }).then((res) => {
-                resolve(res.data)
-            }, (response) => {
-                let urlInfo = {
-                    url: url,
-                    type: 'API_DELETE',
-                    token: token
-                }
-                rejectHandler(response, urlInfo, reject)
-            })
-        })
-    })
-
-}
+  return STORAGE.getAuthToken({ url: url }).then((token) => {
+    return new Promise((resolve, reject) => {
+      let headers = {};
+      if (token) headers['X-CID'] = token;
+      axiosInstance({
+        method: 'delete',
+        url: url,
+        headers,
+      }).then(
+        (res) => {
+          resolve(res.data);
+        },
+        (response) => {
+          let urlInfo = {
+            url: url,
+            type: 'API_DELETE',
+            token: token,
+          };
+          rejectHandler(response, urlInfo, reject);
+        }
+      );
+    });
+  });
+};
 module.exports = {
-    API_GET,
-    API_POST,
-    API_PUT,
-    API_DELETE
-}
+  API_GET,
+  API_POST,
+  API_PUT,
+  API_DELETE,
+};

```

Comment:
Bro Plz check your PR , why all my files that I already merged to master is showing deleted in your PR, check below deleted files, it's all required